tripal_chado_views/tripal_chado_views.module

```diff
@@ -105,21 +105,11 @@ function tripal_chado_views_menu() {
     'weight' => 2,
   );
 
-  $items['admin/tripal/storage/chado/views-integration/export'] = array(
```

Comment:
At one point you could access this page and choose the views integration to export but that solution was clunky and not ideal. I think this should be removed and was simply cruft.